src/main/views/alphabetical-search.njk

```diff
@@ -3,7 +3,7 @@
 
 {% extends "template.njk" %}
 {% block pageTitle %}
-  {{ "Court List" | safe }}
+  {{ title }}
 {% endblock %}
 {% block beforeContent %}
   {{ goBack('/search-option') }}
```

Comment:
Since this PR touches on many files we should probably change back button functionality to `onClick: "history.back();event.preventDefault();"` logic instead

Reply:
Updated to use the new style of back button